Client/Frontend/Browser/Onboarding/OnboardingSearchEnginesViewController.swift

```diff
@@ -40,6 +40,17 @@ class OnboardingSearchEnginesViewController: OnboardingViewController {
         let tablebackground = UIView()
         tablebackground.backgroundColor = OnboardingViewController.colorForTheme(theme)
         contentView.searchEnginesTable.backgroundView = tablebackground
+        
+        // This selection should only ever happen once.
+        // We initially select the default search engine's cell
+        DispatchQueue.main.async {
+            for searchEngine in self.searchEngines.orderedEngines.enumerated() {
+                if searchEngine.element == self.searchEngines.defaultEngine() {
```

Comment:
You can remove this `if` and add it in foreach loop like this
`for searchEngine in self.searchEngines.orderedEngines.enumerated() where searchEngine.element == self.searchEngines.defaultEngine()` {
}

And split in two lines bc it's too long

Reply:
Done.